### ISSUE TITLE
Can I use the picker as the first screen of the app?

### ISSUE THREAD

**Issue by takasurazeem** [Yummypets/YPImagePicker#629]
Can I use it as the first screen of the app with callback functions like `videoDidGetSelected(_ animated) { }`

**Comment by brightroots7** [comment]
Yes, you can create a subclass of `YPImagePicker` thas all, You can get the callbacks on the subclass as well. 

`    
    final class MediaPickerController: YPImagePicker {
      
        private init() {
            super.init(configuration: MediaPickerController.getConfiguration())
        }
        
        required init(configuration: YPImagePickerConfiguration) {
            super.init(configuration: configuration)
        }
        
        required init?(coder aDecoder: NSCoder) {
            fatalError("init(coder:) has not been implemented")
        }
        
        override func viewDidLoad() {
            super.viewDidLoad()
            
            restorationIdentifier = "MediaPickerController"
            restorationClass = MediaPickerController.self
            /* Multiple media implementation */
            super.didFinishPicking { [unowned self] items, cancelled in
                
                if cancelled {
                    print("Picker was canceled")
                    self.dismiss(animated: true, completion: nil)
                    return
                }
                _ = items.map { print("🧀 \($0)") }
                
                let vc = ShareMediaViewController(with: items)
                self.pushViewController(vc, animated: true)
                
            }
    
        }
        
        deinit {
            print("Deinit called #", #file)
        }
    }`

**Comment by takasuratcc** [comment]
`restorationClass = MediaPickerController.self` ```
Cannot assign value of type 'MediaPickerController.Type' to type 'UIViewControllerRestoration.Type?'
Insert ' as! UIViewControllerRestoration.Type' ```


```let vc = ShareMediaViewController(with: items)``` ```<- Cannot find 'ShareMediaViewController' in scope```

I am getting these two errors while attempting to try this solution :/

**Comment by takasurazeem** [comment]
BUMP.
